crates/daemon/src/wallet.rs

```diff
@@ -487,20 +540,46 @@ fn seed_and_rpc_on_same_network(rpc: &electrum_client::Client, network: Network)
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::seed;
+    use bdk::database::MemoryDatabase;
     use bdk_ext::keypair;
     use bdk_ext::new_test_wallet;
     use itertools::Itertools;
+    use rand::distributions::Alphanumeric;
     use rand::thread_rng;
+    use rand::Rng;
     use std::collections::HashSet;
+    use std::env;
+    use std::path::Path;
     use tokio_extras::Tasks;
 
-    impl Actor<(), bdk::database::MemoryDatabase> {
-        pub fn new_offline(
+    const IMPORT_SEED: [u8; 256] = [
+        137, 78, 181, 39, 89, 143, 9, 224, 92, 125, 51, 183, 87, 95, 206, 236, 135, 33, 54, 10,
+        237, 169, 132, 74, 230, 66, 244, 244, 89, 224, 23, 62, 163, 60, 39, 66, 19, 213, 68, 56,
+        125, 14, 244, 247, 52, 68, 200, 153, 127, 86, 77, 119, 239, 133, 59, 129, 112, 250, 77, 36,
+        101, 31, 45, 181, 13, 55, 159, 179, 107, 50, 70, 70, 168, 192, 23, 51, 70, 84, 108, 101,
+        244, 99, 105, 36, 227, 118, 221, 245, 104, 33, 3, 91, 176, 176, 157, 42, 201, 92, 99, 104,
+        235, 84, 237, 229, 131, 204, 227, 15, 223, 109, 8, 47, 155, 132, 236, 108, 106, 59, 185,
+        70, 20, 247, 133, 192, 164, 93, 65, 141, 16, 18, 105, 251, 23, 168, 95, 145, 252, 101, 27,
+        218, 97, 26, 20, 71, 102, 221, 182, 146, 148, 11, 117, 159, 67, 132, 223, 245, 141, 174,
+        157, 226, 67, 5, 61, 218, 98, 131, 8, 170, 233, 140, 139, 198, 24, 107, 60, 147, 236, 194,
+        234, 227, 58, 54, 61, 245, 94, 209, 12, 28, 131, 156, 37, 169, 32, 38, 38, 212, 2, 26, 204,
+        231, 154, 194, 38, 142, 30, 71, 33, 100, 164, 169, 168, 34, 6, 110, 126, 22, 120, 191, 168,
+        219, 64, 70, 172, 224, 218, 142, 135, 179, 79, 140, 174, 157, 60, 43, 141, 218, 178, 45,
+        30, 184, 79, 100, 43, 97, 228, 35, 122, 60, 143, 239, 109, 104, 56, 64, 3, 131,
+    ]; // holds 0.001 btc on testnet
```

Comment:
🤔 Surely this isn't relevant? We should not rely on a testnet address having coins in it for automated testing.